lib/streamlit/elements/vega_charts.py

```diff
@@ -1046,6 +1047,14 @@ def bar_chart(
             Streamlit swaps the x-axis and y-axis and the bars display
             horizontally.
 
+        stack : bool, "normalize", "center", "layered", or None
+            Whether to stack the bars. If this is ``None`` (default), uses Vega's
+            default. If this is ``True``, the bars are stacked on top of each other.
+            If this is ``False``, the bars are displayed side by side. If "normalize",
+            the bars are stacked and normalized to 100%. If "center", the bars are
+            stacked around a central axis. If "layered", the bars are stacked on top
+            of one another.
```

Comment:
Great comment 🙂 What kind of stack is used when simply passing `True`? I find the mix of stack type and boolean a little bit weird tbh. Is this a product requirement? I would rather add another option to the ChartStackType Literal `"default"` or so and test in the code for `None`. This might also make it clearer [here](https://github.com/streamlit/streamlit/pull/8945/files#diff-6ec7e2553ef3075dc705a23fb256495cc04a8eaa1538fc96c9e0bd03bf5466a5R597) where I left the other comment.
What do you think?

Reply:
`stack=True` stacks with baseline at scale's zero value, creating the typical stacked bar chart. 
I could see how having bool & stack type mix would be a bit confusing, though I imagine it's to simplify the most common 2 use cases to `True` (typically expected stacking) & `False` (not stacked). Correct, this API (bool & stack type as string) was outlined in the spec/product requirement.

I may slightly prefer `None` for consistency with the other param defaults and not having to do an additional check for `"default"` + override to None, but not opposed to `"default"` since its probably a bit more clear than `None` 🤔 

_Note: we currently default to vega's 3 point check default system (see [here](https://vega.github.io/vega-lite/docs/stack.html#encoding) if interested)._